Reflection_Engine/Query/MethodList.cs

```diff
@@ -156,9 +156,29 @@ private static void StoreAllMethods(Type type)
 
             if (type.Namespace != null && type.Namespace.StartsWith("BH.") && !type.IsAutoGenerated())
             {
-                m_AllMethodList.AddRange(type.GetMethods(bindingAll).Where(x => x.GetMethodBody() != null && !x.IsAutoGenerated()));
                 m_AllMethodList.AddRange(type.GetConstructors(bindingAll).Where(x => x.GetMethodBody() != null && !x.IsAutoGenerated()));
 
+                MethodInfo[] methods = type.GetMethods(bindingAll);
+                foreach (var method in methods)
+                {
+                    try
+                    {
+                        if (method.GetMethodBody() != null && !method.IsAutoGenerated())
+                            m_AllMethodList.Add(method);
+                    }
+                    catch(Exception e)
+                    {
+                        string message = "Cannot load methods from type  " + type.Name + ". Exception message: " + e.Message;
```

Comment:
The catch is for a single method so the error message should be the same.

Reply:
Not sure I understand, however I corrected the text to say "Cannot load method" (singular, not plural), see commit.
The rest of the text seems fine to me?

Reply:
Thanks but I meant actually telling the user which method cannot be loaded. Right now, they will only know it is from a given type.

Reply:
I wouldn't require a change just for an extra `s` 😋

Reply:
Oh I see! Done!